test/database/src/main/kotlin/no/nav/amt/tiltak/test/database/data/TestDataRepository.kt

```diff
@@ -57,6 +58,18 @@ class TestDataRepository(
 		)
 	}
 
+	fun deaktiverArrangorAnsattRolle(id: UUID) {
+		val sql = """
+			UPDATE arrangor_ansatt_rolle
+			SET gyldig_til = current_timestamp
+			WHERE id = :id
+		""".trimIndent()
+
+		template.update(
+			sql, parameters("id" to id)
+		)
+	}
+
```

Comment:
Føler ikke denne hører helt hjemme her. Bedre å ha den i testen hvor den brukes, eller å bruke insertArrrangorAnsattRolle() til å legge inn en rolle som ikke er gyldig. Alternativt bruke repositoryet som vi allerede har for arrangor_ansatt_rolle